config/linters/.stylelintrc

```diff
@@ -0,0 +1,10 @@
+{
+	"extends": "stylelint-config-standard",
```

Comment:
@aCyborg I added `stylelint-config-standard` which is derived from the common rules found within a handful of CSS styleguides, including: The Idiomatic CSS Principles, Github's PrimerCSS Guidelines, Google's CSS Style Guide, Airbnb's Styleguide, etc. I added this because Daniel's rules were rather thin. We can remove these in favor of a full custom approach if you'd like.